styles.css

```diff
@@ -13,11 +13,12 @@ html {
 body {
   max-width: max-content;
   margin: auto;
+  height: 6715px;
+  border: 1px solid black;
 }
 
 .container {
   width: 375px;
```

Comment:
Kindly, I suggest you not to use fixed width, it is better to use percentages so that your site adapts to all types of mobile devices.